README.md

```diff
@@ -77,7 +77,7 @@ After the request being made, the corresponding response is received in the form
 
 ### Setting the final reference value of the pool
 
-After receiving the response from the Shamba Geospatial Oracle, the final reference value of the corresponding pool is set by calling the `setFinalReferenceValue` function of the [Diva Settlement Facet](https://goerli.etherscan.io/address/0x92c30A4bA4677F5388Aa270087FAb25660648A1D) through the [Diva Diamond Contract](https://goerli.etherscan.io/address/0x2d941518E0876Fb6042bfCdB403427DC5620b2EC) by passing the parameters as `_poolId`, `_finalReferenceValue` having `geostats_data` (18 decimal integer representation) and `_allowChallenge` boolean as `true`.
+After receiving the response from the Shamba Geospatial Oracle, the final reference value of the corresponding pool is set by calling the `setFinalReferenceValue` function of the [Diva Settlement Facet](https://goerli.etherscan.io/address/0x92c30A4bA4677F5388Aa270087FAb25660648A1D) through the [Diva Diamond Contract](https://goerli.etherscan.io/address/0x659f8bF63Dce2548eB4D9b4BfF6883dddFde4848) by passing the parameters as `_poolId`, `_finalReferenceValue` having `geostats_data` (18 decimal integer representation) and `_allowChallenge` boolean as `true`.
```

Comment:
Can you replace the Settlement Facet address `0x92c30A4bA4677F5388Aa270087FAb25660648A1D`  with `0xB17D6c790121D31055dC4C17CA5a8d87413D417e` please?